client/src/components/messenger/Chat.js

```diff
@@ -180,23 +189,20 @@ const Chat = ({ conversationId, title, participants, socket }) => {
             />
             {messages && messages.length > 0 ? (
                 <ul className="chat_section" ref={chatbox}>
-                    {messages.map((msg) => (
+                    {messages.map(msg =>
                         <li
                             key={msg.id}
                             className={"message " + (msg.senderId === id ? "author" : "friend") + "_message"}
                         >
                             <div className="info">
-                                <span className="user" title={msg.sender?.displayName}>
-                                    {msg.sender?.displayName}
-                                </span>
+                                <span className="user" title={msg.sender?.displayName}>{msg.sender?.displayName}</span>
                                 <span className="time">{ChatController.evaluateElapsed(msg.createdAt)}</span>
                             </div>
                             <div className="avatar" href="/">
                                 <img
                                     alt="user avatar"
                                     src={msg.sender?.imageUrl}
                                     title={msg.sender?.displayName}
-                                    width="35"
                                 />
                             </div>
                             <p>{msg.content}</p>
```

Comment:
Can't comment on line 210 but it is causing an error during the pull request checks:

SyntaxError: /home/runner/work/boop/boop/client/src/components/messenger/Chat.js: Unexpected token, expected "}" (210:21)